test_framework/terraform/aws/oracle/user-data-scripts/provision_k3s_agent.sh.tpl

```diff
@@ -1,13 +1,11 @@
 #!/bin/bash
 
-sudo systemctl stop firewalld
-sudo systemctl disable firewalld
-
 sudo yum update -y
 sudo yum group install -y "Development Tools"
 sudo yum install -y iscsi-initiator-utils nfs-utils nfs4-acl-tools cryptsetup
 sudo systemctl -q enable iscsid
 sudo systemctl start iscsid
+sudo systemctl disable nm-cloud-setup.service nm-cloud-setup.timer
```

Comment:
What's the purpose of disabling nm-cloud-setup?

Reply:
Otherwise k3s-agent service won’t start.

Reply:
Can you leave a comment in the above of the change ? Thank you.

Reply:
Comment added. Thank you!